src/EditorFeatures/Core.Cocoa/InlineRename/CommandHandlers/RenameCommandHandler.cs

```diff
@@ -4,18 +4,15 @@
 
 using System;
 using System.ComponentModel.Composition;
-using System.Linq;
-using Microsoft.CodeAnalysis.Editor.Shared.Extensions;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
 using Microsoft.CodeAnalysis.Host.Mef;
-using Microsoft.VisualStudio.Text;
-using Microsoft.VisualStudio.Text.Editor.Commanding;
-using Microsoft.VisualStudio.Text.Operations;
+using Microsoft.VisualStudio.Commanding;
+using Microsoft.VisualStudio.Text.Editor;
 using Microsoft.VisualStudio.Utilities;
-using VSCommanding = Microsoft.VisualStudio.Commanding;
 
 namespace Microsoft.CodeAnalysis.Editor.Implementation.InlineRename
 {
-    [Export(typeof(VSCommanding.ICommandHandler))]
+    [Export(typeof(ICommandHandler))]
     [ContentType(ContentTypeNames.RoslynContentType)]
```

Comment:
> Are we able to move all the MEF metadata attributes down to the base?

We are not. https://github.com/dotnet/roslyn/pull/35465#issuecomment-494110663

Reply:
> Have a shared project that contains RenameCommandHandler and #ifdef's the WPF/Cocoa implementations; that way the attributes are shared.

We're already sharing a bunch of other files in this folder. It would be easy to add this one.